ui/src/router/components/configuration/components/config.js

```diff
@@ -1,7 +1,16 @@
-const getExperimentUrl = (engineProps, experiment) => {
+import template from "lodash/template";
+import templateSettings from "lodash/templateSettings";
+
+export const getHomePageUrl = (engineProps, projectId) => {
+  // If {{projectId}} present in the URL, substitute with actual project id.
+  templateSettings.interpolate = /{{([\s\S]+?)}}/g;
```

Comment:
What does this regex mean?

Reply:
I copied this regex from what was previously added for the client-side formatting of the router Monitoring URL. It's meant to identify variables (which may have whitespaces in between) surrounded by double curly braces `{{ }}`.

In this case, my intention was to match `{{projectId}}` in the URL. As of now, it is the only variable in this scenario.